pkg/apiserver/registry/stats/nodelatencystats/rest_test.go

```diff
@@ -13,3 +13,220 @@
+}
+
+func TestRESTGet(t *testing.T) {
+	tests := []struct {
+		name        string
+		summary     *statsv1alpha1.NodeLatencyStats
+		nodeName    string
+		expectedObj runtime.Object
+		err         error
+	}{
+		{
+			name: "get summary",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			nodeName: "node1",
+			expectedObj: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			err: nil,
+		},
+		{
+			name: "get summary not found",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			nodeName:    "node2",
+			expectedObj: nil,
+			err:         errors.NewNotFound(statsv1alpha1.Resource("nodelatencystats"), "node2"),
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			r := NewREST()
+			ctx := context.Background()
+
+			_, err := r.Create(ctx, tt.summary, nil, nil)
+			require.NoError(t, err)
+
+			obj, err := r.Get(ctx, tt.nodeName, nil)
+			if tt.err != nil {
+				assert.EqualError(t, tt.err, err.Error())
+			} else {
+				require.NoError(t, err)
+				assert.Equal(t, tt.expectedObj, obj)
+			}
+		})
+	}
+}
+
+func TestRESTDelete(t *testing.T) {
+	tests := []struct {
+		name        string
+		summary     *statsv1alpha1.NodeLatencyStats
+		nodeName    string
+		expectedObj runtime.Object
+		err         error
+	}{
+		{
+			name: "delete summary",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			nodeName: "node1",
+			expectedObj: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			err: nil,
+		},
+		{
+			name: "delete summary not found",
+			summary: &statsv1alpha1.NodeLatencyStats{
+				ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+				PeerNodeLatencyStats: nil,
+			},
+			nodeName:    "node2",
+			expectedObj: nil,
+			err:         errors.NewNotFound(statsv1alpha1.Resource("nodelatencystats"), "node2"),
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			r := NewREST()
+			ctx := context.Background()
+
+			_, err := r.Create(ctx, tt.summary, nil, nil)
+			require.NoError(t, err)
+			obj, deleted, err := r.Delete(ctx, tt.nodeName, nil, nil)
+			if tt.err != nil {
+				assert.EqualError(t, tt.err, err.Error())
+			} else {
+				require.NoError(t, err)
+				assert.True(t, deleted)
+				assert.Equal(t, tt.expectedObj, obj)
+			}
+		})
+	}
+}
+
+func TestRESTList(t *testing.T) {
+	summary := &statsv1alpha1.NodeLatencyStats{
+		ObjectMeta:           metav1.ObjectMeta{Name: "node1"},
+		PeerNodeLatencyStats: nil,
+	}
+	options := &internalversion.ListOptions{
+		Limit:    10,
+		Continue: "",
+	}
```

Comment:
remove, we should use `nil` as the list options in this test given that we don't support them yet

Reply:
Yes, it is be better to be removed.